app/Models/Post.php

```diff
@@ -368,4 +368,20 @@ public function scopeWithHiddenPosts($query)
                 ->orWhere('northstar_id', auth()->id());
         }
     }
+
+    /**
+     * Return whether or not a post already has a "good" tag.
+     *
+     * @return bool
+     */
+    public function hasGoodTag()
+    {
+        foreach ($this->tagSlugs() as $tagslug) {
+            if (in_array($tagslug, ['good-for-storytelling', 'good-submission', 'good-for-sponsor', 'good-quote'])) {
```

Comment:
It might be nice to have a property on the `Post` model that returns the allowed tags so that you don't have to define this array in multiple places

Reply:
ah yeah great idea! I knew it was repetitive but couldn't think of how to not repeat myself